Source/Core/Common/SDCardUtil.cpp

```diff
@@ -171,12 +171,12 @@ static void fat_init(u8* fat)
 }
 
 
-static unsigned int write_sector(FILE* file, u8* sector)
+static bool WriteSector(File::IOFile* file, u8* sector)
```

Comment:
Is there any particular reason to use `IOFile*` here over `IOFile&`? IIRC we opted to use the explicit `func(&foo)` syntax (with `R func(P* parameter)`) on caller site to indicate the value is modified rather than making it a reference (which is not distinguishable from a copy). But in this case we don't intend to change it, so would the opposite reasoning make sense?